### PR TITLE
Fixed a bug preventing older versions of Bootstrap from working (introduced by 46e6774)

### PR DESCRIPTION
**Pull request by mmcev106**

### CODE REVIEW

**Comment by djhvscf** [comment]
Please provide a [jsFiddle](https://github.com/wenzhixin/bootstrap-table/blob/develop/CONTRIBUTING.md#bug-reports) to show this PR working, thanks!

**Comment by mmcev106** [comment]
Here's a jsFiddle exhibiting the problem:

https://jsfiddle.net/p2qkh6j8/1/

Notice the 'buttonsClass' javascript error.  Here is a jsFiddle with the problem solved:

https://jsfiddle.net/82m6cfbh/

File: src/bootstrap-table.js [hunk 1]
```diff
@@ -10,7 +10,13 @@
 
   let bootstrapVersion = 3
   try {
-    bootstrapVersion = parseInt($.fn.dropdown.Constructor.VERSION, 10)
+    var rawVersion = $.fn.dropdown.Constructor.VERSION;
+
+    // Only try to parse VERSION if is is defined.
+    // It is undefined in older versions of Bootstrap (tested with 3.1.1).
+    if(rawVersion !== undefined){
```
Comment (djhvscf) [inline src/bootstrap-table.js]: Please add a space between

```javascript
if (rawVersion !== undefined) {
```

**Comment by mmcev106** [comment]
Done.  I also removed the semi-colons since that seemed to be the standard for the rest of the file.

**Pull request merged**